Added a NNID warnings and removed outdated info on Juxt (#375)
PretendoNetwork/website: docs/en_US/install/wiiu.md

# Wii U

> <div class="tip red">
> 	<strong>!!! DISCLAIMER !!!
> 	DON'T REMOVE THE NNID TO SET UP PRETENDO NETWORK, IT'S NOT NECESSARY AND BY DOING SO YOU WILL LOSE ALL YOUR PREVIOUSLY PURCHASED GAMES AND UNABLE TO UPDATE THEM!</strong>
> </div>

Reviewer: ```suggestion
<div class="tip red">
	<strong>CAUTION:</strong>
	DON'T REMOVE YOUR NNID TO SET UP PRETENDO NETWORK, IT'S NOT NECESSARY AND BY DOING SO YOU WILL LOSE ALL YOUR PREVIOUSLY PURCHASED GAMES AND WILL BE UNABLE TO UPDATE THEM!</strong>
</div>
```
